lib/rollbax/item.ex

```diff
@@ -27,7 +27,7 @@ defmodule Rollbax.Item do
     if map_size(meta) == 0 do
       data
     else
-      Map.put(data, "custom", meta)
+      Map.merge(data, meta)
```

Comment:
I think this will break existing code that relies on fact that any provided metadata goes to the `custom` section of payload.
Since top-level keys are predefined we could rely on it.
With something like this:
```elixir
{person, meta} = Map.pop(meta, :person, %{})
{request, meta} = Map.pop(meta, :request, %{})
merge_if_present(data, person)
|> merge_if_present(request)
|> merge_if_present(meta)
```
@manukall WDYT?